feat:support redis username (#76)
auth0/limitd-redis: README.md

`limitd-redis` is client for limits on top of `redis` using [Token Buckets](https://en.wikipedia.org/wiki/Token_bucket).
It's a fork from [LimitDB](https://github.com/limitd/limitdb).

> ## Table of Contents

Reviewer: this is great, is it autogenerated? If so, do we have docs to regen it?

Author: Copilot :D

- `buckets` (object): Setup your bucket types.
- `prefix` (string): Prefix keys in Redis.
- `ping` (object): Configure ping to Redis DB.

> - `username` (string): Redis username. Only to be used in Cluster mode. Needs Redis >= v6.

Reviewer: if not in cluster mode, this is just silently ignored, right?

Do we want to error? If not, maybe we should change this message to:
```suggestion
- `username` (string): Redis username. This is ignored if not in Cluster mode. Needs Redis >= v6.
```

Author: done
